src/usePagination.ts

```diff
@@ -31,12 +31,7 @@ function usePagination<
     ReadonlyArray<$Call<ArrayKeyReturnType<TKey>>> | null,
     PaginationFunction<TOperationType['variables']>,
 ] {
-    const [data, { loadMore, hasMore, isLoading, refetchConnection }] = useOssFragment(
-        fragmentNode,
-        fragmentRef,
-    );
-
-    return [data, { loadMore, hasMore, isLoading, refetchConnection }];
+    return useOssFragment(fragmentNode, fragmentRef);
```

Comment:
Hi @cliedeman, can you explain why this change? 
In this way usePagination is the same as useOssFragment (the `refetch` function must not be available in usePagination)

Reply:
@morrys thats a mistake. I thought I could make this optimization. I will move the memo to the correct place.

Reply:
The change made in useOssFragment seemed correct. I meant the change in usePagination which added the refetch function.